utils/numbers.py

```diff
@@ -0,0 +1,32 @@
+from decimal import Decimal
+from random import randint, uniform
+from typing import Union
+
+DecimalString = Union[Decimal, str]
+
+
+def rounded(
+    value: Union[Decimal, int, float, str], decimals: int = 2, as_string: bool = False
+) -> DecimalString:
+    """
+    Round a decimal to a given amount of decimals and return it as either Decimal or String
+
+    :param value: The number to round
+    :param decimals: The amount of decimals to round value
+    :param as_string: Whether the value should be returned as Decimal or string.
+    For testing, it's useful to be able to parse the Decimal value directly to string.
+    :return: A rounded Decimal number
+    """
+    number = round(Decimal(value), decimals)
+
+    return str(number) if as_string else number
+
+
+def random_int(min: int = 0, max: int = 100, as_string: bool = False) -> DecimalString:
```

Comment:
`random_int` that returns either `Decimal` or `string` :big-hm:

Reply:
if it's not needed yet, maybe we can skip it for now

Reply:
Well, the API tests return `Decimal` values as string, that's why I thought it would be more convenient to have it wrapped in the same function to simplify testing.

Reply:
yeah, but that's why you wrote `random_decimal` 🙂  I mean this util is called `random_int` but does not return an int of really, a bit misleading

Reply:
Well yeah, returns a `Decimal` object which wraps an _integer_ number (i.e. no decimals and no point, e.g. `Decimal("42")`).

If you think it's misleading, we can just leave `random_decimal` and pass `decimals=0` when we'd need an "int".

Reply:
I think we do not need any Decimals in random int generation logic at all. just the following would suffice:
```python
def random_int(min: int = 0, max: int = 100, as_string: bool = False) -> Union[Int, String]:
    integer = randint(min, max)
    return str(integer) if as_string else integer
```

Reply:
In that case, yes. In any case, it's not being used at the moment 😛 so we can just pretend this conversation never happened.